editor/editor_audio_buses.cpp

```diff
@@ -644,18 +651,43 @@ Variant EditorAudioBus::get_drag_data_fw(const Point2 &p_point, Control *p_from)
 
 bool EditorAudioBus::can_drop_data_fw(const Point2 &p_point, const Variant &p_data, Control *p_from) const {
 	Dictionary d = p_data;
-	if (!d.has("type") || String(d["type"]) != "audio_bus_effect") {
+
+	if (!d.has("type")) {
 		return false;
 	}
 
-	TreeItem *item = effects->get_item_at_position(p_point);
-	if (!item) {
-		return false;
+	if (String(d["type"]) == "audio_bus_effect") {
+		TreeItem *item = effects->get_item_at_position(p_point);
+		if (!item) {
+			return false;
+		}
```

Comment:
This check should also be done outside this block. Otherwise dropping a file below effects results in a no op.
![godot windows tools x86_64_R8zeStSnRm](https://user-images.githubusercontent.com/2223172/191491911-43332cbc-7c78-495f-a462-9086cb9b0f98.gif)

Reply:
Fixed